src/overlay/Peer.h

```diff
@@ -193,6 +197,8 @@ class Peer : public std::enable_shared_from_this<Peer>,
     VirtualClock::time_point mCreationTime;
 
     VirtualTimer mRecurringTimer;
+    VirtualTimer mEarlyCatchupTimer;
```

Comment:
I know this was here before, but could we rename this variable? This timer triggers delayed execution of a function, and in our case the function then _maybe_ triggers another process, i.e. "early catchup". I think we can simply name the variable after what it does, `executeDelayed` or `scheduleExecution` or something like that.

Reply:
Named to it mExecutionDelayedTimer.